### ISSUE TITLE
Eclipse Oxygen + Buildship 2.0.2 compatibility

### ISSUE THREAD

**Issue by oliverwelter** [herkne/buildship-import-4-oomph#2]
Hi,
in the latest release of Eclipse Oxygen Buildship 2.0.2 is used which prevents the buildship import plugin to install because of version incompatibility.

Best,
Oliver

**Comment by herkne** [comment]
Yes, you are correct. But due to some api changes the plugin does not compile any longer.
I don't see the necessity for updating the plugin - the 'project import' task works fine for some time now.  And also considering my limited free time I do not think, that I will provide an update in the foreseeable future.
Best regards,
Hermann

**Comment by oliverwelter** [comment]
Could you please provide an example how you used the 'project import' task for importing a gradle project via buildship. I would like to avoid to checkin the Eclipse project files into the source repository.
Thanks.

Best regards,
Oliver
